packages/Tabs/src/Tabs.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import PropTypes from "prop-types";
+import TabsContext from "./TabsContext";
+import Panel from "./components/Panel/Panel";
+import Panels from "./components/Panels/Panels";
+import Tab from "./components/Tab/Tab";
+import List from "./components/List/List";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  defaultIndex: PropTypes.number,
+  isDisabled: PropTypes.bool,
+};
+
+const defaultProps = {
+  defaultIndex: 0,
+  isDisabled: false,
+};
+
+const Tabs = props => {
+  const [activeIndex, setActiveIndex] = React.useState(props.defaultIndex);
+  const [currentFocusIndex, setFocusIndex] = React.useState(null);
+
+  let tabListRef = React.useRef(null);
+
+  const { isDisabled } = props;
+
+  function focusAndSetIndex(index) {
+    tabListRef.querySelectorAll(".tab")[index].focus();
+    setFocusIndex(index);
```

Comment:
This is for manual activation, We don't want to use automatic activation for tabs